charts/intel-gpu-resource-driver/values.yaml

```
@@ -0,0 +1,85 @@
+# Default values for intel-gpu-resource-driver.
+nameOverride: ""
+namespaceOverride: ""
+fullnameOverride: ""
+selectorLabelsOverride: {}
+
+crds:
+  enabled: true
+
+imagePullSecrets: []
+image:
+  repository: intel
+  name: intel-gpu-resource-driver
+  pullPolicy: Always
```

```suggestion
  pullPolicy: IfNotPresent
```
For development purposes pullPolicy `Always` is good, but for release we don't need the same image pulled all the time, let it be "IfNotPresent"